### ISSUE TITLE
Dropdown trigger generate warning

### ISSUE THREAD

**Issue by ghost** [Semantic-Org/Semantic-UI-React#676]
``` javascript
render() {
    const TRIGGER = (
        <div className="user-info">
          <div className="round-icon">
            <i className="fa fa-user"></i>
          </div>
          <div className="user-name">
            <span>Hi {this.props.user.getFullName()}</span>
          </div>
        </div>
    );

    return (
        <Dropdown trigger={TRIGGER} pointing="top left" icon={null}>
          <Dropdown.Menu>
            <Dropdown.Item text="Profile" />
            <Dropdown.Item text="Log out" />
          </Dropdown.Menu>
        </Dropdown>
    );
  }
```

generate next warning

```
warning.js:36 Warning: Failed prop type: `children` prop in `Dropdown` requires props: `text`.
    in Dropdown (created by UserInfo)
    in UserInfo (created by Connect(UserInfo))
    in Connect(UserInfo) (created by AppHeader)
    in div (created by AppHeader)
    in div (created by AppHeader)
    in AppHeader (created by Connect(AppHeader))
    in Connect(AppHeader) (created by App)
    in div (created by App)
    in App (created by Connect(App))
    in Connect(App) (created by RouterContext)
    in RouterContext (created by Router)
    in Router
    in Provider
```

any suggestions how to fix this?

**Comment by jeffcarbs** [comment]
@SergeyShablenko - there's some work in progress to refactor the dropdown component #672. That PR will also address these proptype warnings so we'll leave this open until that's merged.

**Comment by levithomason** [comment]
This warning is also fixed in [#645 (Dropdown.js)](https://github.com/Semantic-Org/Semantic-UI-React/pull/645/files#diff-609214949026e594dbfd98481664c928).  Once merged, it will go away.

**Comment by levithomason** [comment]
@SergeyShablenko this is now fixed in `semantic-ui-react@0.56.4`, cheers!
